usbhub.cpp

```diff
@@ -104,6 +105,14 @@ uint8_t USBHub::Init(uint8_t parent, uint8_t port, bool lowspeed) {
         if(!bAddress)
                 return USB_ERROR_OUT_OF_ADDRESS_SPACE_IN_POOL;
 
+        {
+                UsbDeviceAddress a;
+                a.devAddress = bAddress;
+                a.bmHub = 0;
+                a.bmAddress = port ? : 1;
```

Comment:
Not sure what this does when port is non-zero. Is it the equivalent of
if (port) {
 a.bmAddress = ;
} else {
 a.bmAddress = 1;
}

Reply:
Yes. Port numbers start at 1.
Right now commits are stopped, but eventually should be resumed.